packages/docs/pages/page.tsx

```diff
@@ -0,0 +1,287 @@
+import { H1, Panel, Text } from '@bigcommerce/big-design';
+import { AddIcon, ArrowDropDownIcon } from '@bigcommerce/big-design-icons';
+import { ActionBar, Header, Page } from '@bigcommerce/big-design-patterns';
+import React, { Fragment } from 'react';
+
+import {
+  Code,
+  CodePreview,
+  CodeSnippet,
+  ContentRoutingTabs,
+  GuidelinesTable,
+  List,
+} from '../components';
+import { BackgroundPropsTable, PagePropsTable } from '../PropTables/PagePropTable';
+
+const HeaderPage = () => {
+  return (
+    <>
+      <H1>Page</H1>
+
+      <Panel header="Overview" headerId="overview">
+        <Text>
+          The <Code primary>Page</Code> component serves as a high-level layout component that
+          organizes the content of a page, including optional elements such as a header, message,
```

Comment:
Since we are referring to props/components (`Header` `message`, and `ActionBar`), can we wrap them in a `<Code />` component.

Reply:
Thanks @chanceaclark , this recommendation ahs been added